Geodata_CityGML/Geodata_CityGML_ontology.ttl

```diff
@@ -0,0 +1,17 @@
+@prefix Geodata_CityGML: <https://github.com/GAIA-X4PLC-AAD/map-and-scenario-data/tools/ontologie_creator/ontologies/Geodata_CityGML_ontology/> .
```

Comment:
prefix and directory name should follow the guidelines.

--> geodata-city-gml
both directory and prefix should match.